optimize/backend/src/main/java/io/camunda/optimize/license/LicenseType.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.license;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class is a duplicate of src/main/java/io/camunda/service/license/LicenseType.java
+ *
+ * <p>This class exists because Optimize is not part of the single application, and cannot use any
+ * of the monorepo's modules. Once Optimize is added, the `service` implementation of `LicenseType`
+ * can be used, and this Optimize duplicate can be removed
+ */
+public enum LicenseType {
+  SAAS("saas"),
+  PRODUCTION("production"),
+  UNKNOWN("unknown");
+  private static final Map<String, LicenseType> ENUM_MAP;
+
+  static {
+    final Map<String, LicenseType> map = new HashMap<>();
+    for (final LicenseType instance : LicenseType.values()) {
+      map.put(instance.getName().toLowerCase(), instance);
+    }
+    ENUM_MAP = Collections.unmodifiableMap(map);
+  }
```

Comment:
I feel like all of this is redundant and could be better handled with a much simplified `get` implementation. Example:

```
public static LicenseType get(final String name) {
  Arrays.stream(LicenseType.values())
    .filter(e -> e.name().equalsIgnoreCase(dayName)).findAny().orElse(LicenseType.UNKNOWN);
}
```

Reply:
I see what you're doing here, and agree - this is simpler. The `ENUM_MAP` here is a overkill for something that isn't called a whole lot and doesn't have a lot of `enum` values